prac_10/testing.py

```diff
@@ -0,0 +1,90 @@
+"""
+CP1404/CP5632 Practical
+Testing demo using assert and doctest
+"""
+
+import doctest
+from prac_06.car import Car
+
+
+def repeat_string(input_string, number_of_times_repeated):
+    """Repeat input_string a given amount of times with spaces in between."""
+    repeated_strings = []
+    for i in range(number_of_times_repeated):
+        repeated_strings.append(input_string)
+    output_string = " ".join(repeated_strings)
+    return output_string
+
+
+def is_long_word(word, length=5):
+    """
+    Determine if the word is as long or longer than the length passed in
+    >>> is_long_word("not")
+    False
+    >>> is_long_word("supercalifrag")
+    True
+    >>> is_long_word("Python", 6)
+    True
+    """
+    return len(word) >= length
+
+
+def format_phrase_as_sentence(phrase: str):
+    """
+    Format phrase as a sentence with capital letter and period.
+    >>> format_phrase_as_sentence('hello')
+    'Hello.'
+    >>> format_phrase_as_sentence('It is an ex parrot.')
+    'It is an ex parrot.'
+    >>> format_phrase_as_sentence('Nice to meet you')
+    'Nice to meet you.'
+    """
+    sentence = ""
+    if phrase[0].islower():
```

Comment:
Can this if statement be simplified by capitalizing the sentence regardless? We know that it needs to be a capital, so rather than test for it and then make it a capital, assume it always starts with lowercase and always change it to capital.